docs/REDUX_STRUCTURE.md

```diff
@@ -0,0 +1,118 @@
+# Redux Multi Currencies Structure
+
+## Store structure
+For being able to store multi currencies data, we should need a store structure more similar to the one at [lisk-mobile](https://github.com/liskHQ/lisk-mobile).
+
+### Lisk Mobile
+Showing here just the parts related to the multi currencies structure.
+```json
+// lisk-mobile store example
+{
+  "..."
+  "accounts": {
+    "passphrase": "",
+    "followed": { "tokenKey": [{}] },
+    "info": { "tokenKey": { "address": "", "balance": "", "publickey": "", "..." } }
+  },
+  "service": {},
+  "settings": {
+    "token": {
+      "active": "tokenKey",
+      "list": { "tokenKey": true }
+    },
+    "..."
+  }
+}
+```
+Where we have the accounts information and the followed accounts separated per token type, and other informations that are relevant for more than one token being directly on the `accounts` node.  
+On `settings` we have which token is the active one, and also a list with the tokens, of which token is enabled by the user.  
+And also a unified `service` node, not just as liskService as we have right now on Lisk-Hub.  
+
+### Lisk Hub
+In the case of Lisk-Hub we could have initialy a structure that integrate some of the changes just for other currencies, while we don't update the current LSK structure.  
+To have a better structure for new currencies, while also keeping the LSK token working without too much work, something like:
+```json
+// lisk-hub intermediate store example
+{
+  "..."
+  "account": { "LSK account info" },
+  "accounts": { "info": { "BTC": {} }, "followed": {} },
+  "BTC": { "BTC specific data" },
+  "service": {},
+  "delegate": {},
+  "voting": {},
+  "filters": {},
+  "..."
+}
+```
+- Rename `liskService` to just `service` so we can have all service information centralized in one place.  
+- New token accounts being already on the `accounts` structure.
+- Specific new token data grouped inside a `tokenKey` node.
+
+Ideally in the future we should end up with a structure similar to:
+```json
+// lisk-hub final store example
+{
+  "accounts": { "info": { "tokenKey": {} }, "followed": {}, "passphrase": "", "other account common info" },
```

Comment:
IMO it's better to make the `followed` an array of objects with `tokenKey` property. We can always filter `followed` by `tokenKey` in situations where we want to display only accounts of one `tokenKey`

With the current proposal of `followed` being an object with `tokenKey` keys and separate lists of accounts, it will be impossible to have a single list all tokens together with a given order where user can have their most important accounts from all tokens at the top, as planned here:
https://netguru.invisionapp.com/share/M5RKJHPQZKF#/screens/357779158

With the structure from Lisk Mobile, it would mean that the list has all LSK accounts and only after that all BTC accounts (or vice versa).

```suggestion
  "accounts": { "info": { "tokenKey": {} }, "followed":  [{ "tokenKey": "...", ...}, ...], "passphrase": "", "other account common info" },
```

Reply:
I also see a problem of `followed` being inside the `accounts` object, because unlike in Mobile, in Hub the followed accounts are not linked to the account and can be accessed on the dashboard in the guest mode when there is no user at all. Keeping `followed` inside `user` will make reducers for actions like `login` and `logout` more complicated because they will have to preserve this part of `account` store.

Reply:
So something like this would be better? @slaweet 
```suggestion
  "accounts": { "info": { "tokenKey": {} }, "passphrase": "", "other account common info" },
  "followed": [],
```

Reply:
I think is a real good point what Vit mentioned, for the follow accounts we can handle it separated from accounts same for the array of objects with the tokenKey it will be better as Vit mentioned